gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/rebind/ClassInspector.java

```diff
@@ -261,9 +261,10 @@ public void collectStaticAnnotatedFields(JClassType type,
                 if (field.getAnnotation(annotation) != null) {
                     JParameterizedType parameterizedType = field.getType().isParameterized();
                     if (!field.isStatic()
-                            || parameterizedType == null
-                            || !type.isAssignableFrom(parameterizedType)
-                            || !typeParameter.isAssignableFrom(parameterizedType.getTypeArgs()[0])) {
+                             || (typeParameter != null
```

Comment:
I'm not sure why this is here.  Sorry I wrote all this 4 months ago. I'll double check, I think it probably isn't needed.